content/account_management/faq/api-app-key-management.md

```diff
@@ -0,0 +1,38 @@
+---
+title: API and APP Key Management
+kind: faq
+aliases:
+    - /account_management/faq/how-do-i-reset-my-application-keys/
+    - /agent/faq/how-do-i-reset-my-datadog-api-keys/
+---
+
+## API keys
+
+API keys are unique to your organization. An API key is required by the Datadog Agent to submit metrics and events to Datadog.
+
+## APP keys
+
+Application keys, in conjunction with your org's API key, give you full access to Datadog's programmatic API. Application keys are associated with the user account that created them and must be named. The application key is used to log all requests made to the API.
+
+## Add
+
+To add a Datadog API or APP key, navigate to [Integration -> APIs][1], enter a name for your key, and click **Create API key** or **Create Application Key**.
+
+**Note**:
+
+* Your org must have at least one API key and at most API five keys.
```

Comment:
```suggestion
* Your org must have at least one API key and at most five API keys.
```

Should we say that they can reach out to support for more?

Reply:
I've asked this in the past and they didn't want to advertise there was a feature for this.